sharness/install-dvc.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+virtualenv --quiet --python python3 .env
+source .env/bin/activate
+#pip3 install --quiet --editable "..[all]"
+pip3 install --quiet --editable ".."
```

Comment:
Why test the dev version of DVC and not the latest stable release?

Reply:
> Why test the dev version of DVC and not the latest stable release?

The automated tests are usually run after you make some changes (commits or PR) to make sure that they don't break anything. That's why you usually want the latest version.

But if these tests are not going to be used by the developers (which is the case if they are moved to another repository), then probably it makes more sense to use the latest (stable) release.

Besides this, the automated tests don't show you the output and interaction of the commands. If we move the tests to another repository (which is intended to be used by the "testers", not by the developers) it would make more sense to keep them in the form of simple normal bash scripts (as opposed to automated sharness tests), which can be run manually and allow you to inspect as well the output and interaction of the commands.

Reply:
Ah, good points. So if they stay in this repo then yes, install and test the latest version or "dev mode".